common/src/main/java/com/microsoft/identity/common/internal/providers/microsoft/MicrosoftAuthorizationRequest.java

```diff
@@ -89,6 +91,9 @@
     @SerializedName("x-client-DM")
     private String mDiagnosticDM;
 
+    protected transient AzureActiveDirectorySlice mSlice;
+
+    protected transient HashMap<String, String> mFlightParameters;
```

Comment:
Does the concrete class matter here? Or can this be declared as `Map<String, String>`?

Reply:
Fixed.